fanuc_cr7ia_support/urdf/cr7ial_macro.xacro

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:property name="pi" value="3.1415926535897931" />
```

Comment:
If you start `xacro` with the `--inorder` command line option (which I added) then it uses Jade+ xacro.